app/models/service/aggregation.rb

```diff
@@ -315,7 +315,7 @@ def aggregate_all_vm_disk_space_used
   def aggregate_all_vm_memory_on_disk
     if has_attribute?("aggregate_all_vm_memory_on_disk")
       # Avoids (poorly) "ERROR:  integer out of range" in postgres
-      self["aggregate_all_vm_memory_on_disk"] * 1.megabyte
+      self["aggregate_all_vm_memory_on_disk"].try(:*, 1.megabyte)
```

Comment:
Seems like it would return the proper expected output from the other columns (`nil` if the column doesn't return a result in that query), so looks good.  👍 

In retrospect though, looking at the `else` case, we probably should have written the SQL to include a `COALESCE(SUM('INSERT_COLUMN_HERE'), 0)` on all of these, or defaulted them to zero, since that is the defaulted in the `else` case as well.  But that is out of scope for what we are trying to do here, so no need to handle that now.